src/config/routes.rb

```diff
@@ -1,8 +1,14 @@
 Sessionizer::Application.routes.draw do
   root to: 'schedules#index'
 
-  get '/home' => 'pages#home', as: :home_page
+  get '/home' => 'events#show', id: 'current', as: :home_page
 
+  # Session actions available for all events
+  resources :events do
```

Comment:
Do we still need `resources :events, only: [:show] do`?

Reply:
I think you're right: we're only using `show` right now. And probably will only want to add `index` in the future.